.github/workflows/build-and-test-dotnet.yml

```diff
@@ -158,7 +158,7 @@ jobs:
           token: ${{ secrets.CHECKOUT_TOKEN }}
 
       - name: Set up .NET
-        uses: Lombiq/GitHub-Actions/.github/actions/setup-dotnet@dev
```

Comment:
Also update the default .NET version in this workflow.

Reply:
Thanks. I reverted the change in this file accidentally.